### ISSUE TITLE
Whitelisting unscoped items

### ISSUE THREAD

**Issue by bobbbay** [neoeinstein/aliri#17]
I'm using `aliri-axum`/`aliri-tower`, and can't find a way to "whitelist" a route, in order for it to stop checking for authentication. This is especially useful for a "login" route, but there's more than one route in my application that should be public to those unauthorized. However, even if I completely omit the scope from a handler's signature, it still returns `aliri_tower::OnJwtError::on_missing_or_malformed`.

In other words, I would like to be able to open some - but not all - routes to people without bearer tokens. Am I missing how to do this?

**Comment by neoeinstein** [comment]
You may be able to do this with `Router::merge` and using two routers, one for authenticated routes, and another for unauthenticated routes. Take a look at the code I've added to the Auth0 example here: https://github.com/neoeinstein/aliri/pull/18/files

Let me know if that solves your problem.

**Comment by bobbbay** [comment]
Yes, it does, thank you very much.
